models/rate_transformer_node_impl.h

```diff
@@ -66,7 +66,6 @@ template < class TNonlinearities >
 nest::rate_transformer_node< TNonlinearities >::Parameters_::Parameters_()
   : linear_summation_( true )
 {
-  recordablesMap_.create();
```

Comment:
why was this removed?

Reply:
I removed it here because it is also called in the constructor for the neuron itself, and that is the canonical place to call it.